packages/image_picker/lib/image_picker.dart

```diff
@@ -56,15 +56,24 @@ class ImagePicker {
     return path == null ? null : new File(path);
   }
 
+  /// Returns a [File] object pointing to the video that was picked.
+  ///
+  /// The [source] argument controls where the video comes from. This can
+  /// be either [ImageSource.camera] or [ImageSource.gallery].
+  ///
+  /// [maxDuration] is the maximum amount of Seconds (eg. 30.0) that is available to be recorded
```

Comment:
What happens if you select a video that's longer than `maxDuration` from the gallery? Is the video shortened to the provided duration? Will you be unable to select longer videos? This would be good to document.